src/config.rs

```diff
@@ -1019,6 +1031,38 @@ impl Config {
     }
 }
 
+/// Window Dimensions
+///
+/// Initial window dimensions in characters
+#[derive(Debug, Deserialize)]
+pub struct Dimensions {
```

Comment:
It might make sense to make this a newtype on `index::Point` which is a struct with `lines` and `cols`.